docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -241,26 +241,53 @@ In order to use a Nessie catalog, ensure to configure the catalog type with
 properties:
 
 .. list-table:: Nessie catalog configuration properties
-  :widths: 40, 60
+  :widths: 30, 58, 12
   :header-rows: 1
 
   * - Property name
     - Description
+    - Default
   * - ``iceberg.nessie-catalog.uri``
     - Nessie API endpoint URI (required).
       Example: ``https://localhost:19120/api/v1``
+    -
   * - ``iceberg.nessie-catalog.ref``
-    - The branch/tag to use for Nessie, defaults to ``main``.
+    - The branch/tag to use for Nessie.
+    - ``main``
   * - ``iceberg.nessie-catalog.default-warehouse-dir``
     - Default warehouse directory for schemas created without an explicit
       ``location`` property. Example: ``/tmp``
+    -
+  * - ``iceberg.nessie-catalog.authentication.type``
+    - The authentication type to use.
+      Available values are ``NONE`` or ``BEARER``.
+      Example: ``BEARER``
+    - ``NONE``
+  * - ``iceberg.nessie-catalog.authentication.token``
+    - The token to use with ``BEARER`` authentication.
+      Example: ``SXVLUXUhIExFQ0tFUiEK``
+    -
+  * - ``iceberg.nessie-catalog.read-timeout``
+    - The read timeout duration for requests to the Nessie server.
+      Example: ``5s``
+    - ``25s``
+  * - ``iceberg.nessie-catalog.connection-timeout``
+    - The connection timeout duration for connection requests to the Nessie server.
+      Example: ``1m``
+    - ``5s``
+  * - ``iceberg.nessie-catalog.compression-enabled``
+    - Configure whether compression should be enabled or not for
+      requests to the Nessie server.
+    - ``true``
 
 .. code-block:: text
 
     connector.name=iceberg
     iceberg.catalog.type=nessie
     iceberg.nessie-catalog.uri=https://localhost:19120/api/v1
     iceberg.nessie-catalog.default-warehouse-dir=/tmp
+    iceberg.nessie-catalog.authentication.type=BEARER
```

Comment:
Let's keep things simple for the users wanting to try out Nessie.
Leave the authentication details out of the sample config for Nessie catalog.

Reply:
ack